plugins/main/public/components/common/wazuh-discover/wz-flyout-discover.tsx

```diff
@@ -144,7 +142,7 @@ const WazuhFlyoutDiscoverComponent = (props: WazuhDiscoverProps) => {
     setIndexPattern(dataSource?.indexPattern);
     fetchData({
       query,
-      dateRange: absoluteDateRange,
+      dateRange: { from: dateRange.from || '', to: dateRange.to || '' },
```

Comment:
**question**: could you elaborate the reason of the default value (empty string)?